rules/coding-style/src/Rector/Throw_/AnnotateThrowablesRector.php

```diff
@@ -421,6 +427,14 @@ private function reflectionFunctionGetNode(ReflectionFunction $reflectionFunctio
         return $ast;
     }
 
+    private function getClassFromMethodCall(MethodCall $methodCall):?FullyQualified
+    {
+        // $variable = new Class()
+        $class = $methodCall->getAttribute('previousExpression')->expr->expr->class;
+
+        return $class instanceof FullyQualified ? $class : null;
+    }
```

Comment:
This looks very magical and unrealiable... What is the purpose?

```php
$staticType = $this->getStaticType($variableNode)
if ($staticType instanceof ObjectType) {
    // ...
}
```

Reply:
Yes, but maybe it is something forgotten from the first version... Anyway, now I removed it...

Reply:
No troubles. Maybe I was too eager to review :D 

Just ping me next time, when you'll need my feedback.